package.json

```
@@ -45,7 +45,7 @@
     "gatsby-remark-check-links": "^2.1.0",
     "gatsby-remark-copy-linked-files": "^2.3.2",
     "gatsby-remark-images": "^5.0.0",
-    "gatsby-remark-prismjs": "^3.5.1",
+    "gatsby-remark-prismjs": "^4.0.0",
```

this at least would represent the version getting installed
```suggestion
    "gatsby-remark-prismjs": "^4.2.0",
```

---

it's fine, I'm just trying to get this stuff bumped to get rid of the not-made-for-gatsby 3 warnings so I can then bump gatsby to 4